### DISCUSSION TITLE
How to secure a section of the docs in a simple way

### DISCUSSION THREAD

**Discussion by e4rthdog** [squidfunk/mkdocs-material#6939] [Q&A] [answered]
The need is to be able to secure a sub tree from the documentation. It can be a top level directory appearing in the NAV, or a sub tree

The only simple solution that  i will try is to use the web servers capabilities on basic auth (Apache). I dont need usernames, simple password should be ok. The only requirement is not to ask the password all over again when we visit pages inside the tree.

Are there any other proposals? The doc site is internal to the organization and cannot be hosted outside.

**Comment by alexvoss** [comment, marked as answer]
See the discussion here: https://github.com/squidfunk/mkdocs-material/discussions/5050

Yes, securing your site or parts thereof is the duty of the webserver. There is nothing a static site generator needs to do or can do in this regard, at least nothing generic. If you wanted to show who is logged in, for example, you would need to use [customization](https://squidfunk.github.io/mkdocs-material/customization/) with some extra JavaScript.
